core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -458,7 +458,7 @@ class KafkaServer(val config: KafkaConfig, time: Time = Time.SYSTEM, threadNameP
             catch {
               case ioe: IOException =>
                 ioException = true
-                warn("Error during controlled shutdown, possibly because leader movement took longer than the configured socket.timeout.ms: %s".format(ioe.getMessage))
+                warn("Error during controlled shutdown, possibly because leader movement took longer than the configured controller.socket.timeout.ms or request.timeout.ms: %s".format(ioe.getMessage))
```

Comment:
Thanks for the review @ijuma. Agree, `and/or` makes more sense. Will update the message.